packages/@romejs/core/common/utils/executeMain.ts

```diff
@@ -51,7 +51,9 @@ export default async function executeMain(
     setImmediate,
     setInterval,
     setTimeout,
-    require: internalModule.createRequireFromPath(filename),
+    require:
+      internalModule.createRequire(filename) ||
+      internalModule.createRequireFromPath(filename),
```

Comment:
I think what you meant was

```suggestion
      internalModule.createRequire === undefined ? internalModule.createRequireFromPath(filename) : internalModule.createRequire(filename),
      
```

Reply:
Ups, yes indeed! 😓 I've changed it, thanks for finding this.